Tools/autotest/sim_vehicle.py

```diff
@@ -719,7 +728,7 @@ def start_mavproxy(opts, stuff):
                "you are simulating, for example, start in the ArduPlane directory to " \
                "simulate ArduPlane")
 
-parser.add_option("-v", "--vehicle", type='string', default=None, help="vehicle type (ArduPlane, ArduCopter or APMrover2)")
+parser.add_option("-v", "--vehicle", type='choice', default=None, help="vehicle type (%s)" % vehicle_options_string, choices=default_frame_for_vehicle.keys())
```

Comment:
Should the parenthesis be removed in the help text? I think that `(<APMrover2|ArduPlane|ArduCopter|AntennaTracker>)` looks weird.
Maybe removing the less/bigger characters in *vehicle_options_string* and only adding them where needed is a better option. Also I think that putting spaces (so using `' | '`) would look better: `APMrover2 | ArduPlane | ArduCopter | AntennaTracker`